redis/src/main/scala/zio/redis/api/Keys.scala

```diff
@@ -216,6 +216,57 @@ trait Keys {
     `type`: Option[String] = None
   ): ZIO[RedisExecutor, RedisError, (Long, Chunk[String])] = Scan.run((cursor, pattern, count, `type`))
 
+  /**
+   * Sorts the list, set, or sorted set stored at key. Returns the sorted elements.
+   *
+   * @param key key
+   * @param by by option, specifies a pattern to use as an external key
+   * @param limitOffset limit count and offset option, take only limit values, starting at position offset
+   * @param desc desc option, sort descending instead of ascending
+   * @param get get option, return the values referenced by the keys generated from the get patterns
+   * @param alpha alpha option, sort the values alphanumerically, instead of by interpreting the value as floating point number
+   * @return the sorted values, or the values found using the get patterns
+   */
+  final def sort(
+    key: String,
+    by: Option[String] = None,
+    limitOffset: Option[(Int, Int)] = None,
+    desc: Boolean = false,
```

Comment:
Order is defined as geo options [here](https://github.com/zio/zio-redis/blob/b72e4fc4b98493baf5ae782868517cbedfdd9369/redis/src/main/scala/zio/redis/options/Geo.scala#L11). Please do the same approach:
- move to shared
- use it instead of plain boolean